docs/shared/reference/project-configuration.md

```diff
@@ -158,6 +158,19 @@ Examples:
 
 Note the result value is hashed, so it is never displayed.
 
+_External Dependencies_
+
+This input is useful for ambiguous executors like `nx:run-commands`.
```

Comment:
This needs more explanation.  Something like

```suggestion
This input is useful for executors that do not have code directly associated with them, like `nx:run-commands`.
```

Reply:
Maybe that's not the right explanation, but whatever the right explanation is needs to go here.